Mage.Sets/src/mage/cards/f/ForgeAnew.java

```diff
@@ -0,0 +1,146 @@
+package mage.cards.f;
+
+import java.util.HashSet;
+import java.util.Set;
+import java.util.UUID;
+
+import mage.abilities.Ability;
+import mage.abilities.common.EntersBattlefieldTriggeredAbility;
+import mage.abilities.common.SimpleStaticAbility;
+import mage.abilities.condition.common.MyTurnCondition;
+import mage.abilities.decorator.ConditionalAsThoughEffect;
+import mage.abilities.effects.common.ReturnFromGraveyardToBattlefieldTargetEffect;
+import mage.abilities.effects.common.continuous.ActivateAbilitiesAnyTimeYouCouldCastInstantEffect;
+import mage.abilities.effects.common.cost.CostModificationEffectImpl;
+import mage.abilities.keyword.EquipAbility;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.*;
+import mage.filter.FilterCard;
+import mage.game.Game;
+import mage.game.events.GameEvent;
+import mage.game.stack.StackObject;
+import mage.players.Player;
+import mage.target.common.TargetCardInYourGraveyard;
+import mage.watchers.Watcher;
+
+/**
+ *
+ * @author xanderhall
+ */
+public class ForgeAnew extends CardImpl {
+
+    private static final FilterCard filter = new FilterCard("Equipment card from your graveyard");
+
+    static {
+        filter.add(SubType.EQUIPMENT.getPredicate());
+    }
+
+    public ForgeAnew(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.ENCHANTMENT}, "{2}{W}");
+
+        // When Forge Anew enters the battlefield, return target Equipment card from your graveyard to the battlefield.
+        Ability ability = new EntersBattlefieldTriggeredAbility(new ReturnFromGraveyardToBattlefieldTargetEffect());
+        ability.addTarget(new TargetCardInYourGraveyard(filter));
+        this.addAbility(ability);
+
+        //As long as it’s your turn, you may activate equip abilities any time you could cast an instant.
+        this.addAbility(new SimpleStaticAbility(new ConditionalAsThoughEffect(
+            new ActivateAbilitiesAnyTimeYouCouldCastInstantEffect(EquipAbility.class, "equip abilities"), MyTurnCondition.instance
+            ).setText("as long as it's your turn, you may activate equip abilities any time you could cast an instant."))
+        );
+
+        //You may pay {0} rather than pay the equip cost of the first equip ability you activate during each of your turns.
+        this.addAbility(new SimpleStaticAbility(new ForgeAnewCostEffect()), new ForgeAnewWatcher());
+    }
+    private ForgeAnew(final ForgeAnew card) {
+        super(card);
+    }
+
+    @Override
+    public ForgeAnew copy() {
+        return new ForgeAnew(this);
+    }
+}
+
+class ForgeAnewCostEffect extends CostModificationEffectImpl {
+
+    ForgeAnewCostEffect() {
+        super(Duration.WhileOnBattlefield, Outcome.Benefit, CostModificationType.SET_COST);
+        this.staticText = "you may pay {0} rather than pay the equip cost of the first equip ability you activate each turn.";
```

Comment:
Not identical to Bruenor, this should be "each of your turns" and somewhere check it's your turn in case something else is letting you activate equip abilities at instant speed

Reply:
I missed that difference in wording, I'll take a look!